examples/cli/src/main/java/io/unitycatalog/cli/utils/FileOperations.java

```diff
@@ -0,0 +1,84 @@
+package io.unitycatalog.cli.utils;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+
+public class FileOperations {
+
+  /**
+   * This helper function adjusts local file URI that starts with file:/ or file:// but not with
+   * file:///. This function makes sure these URIs must begin with file:/// in order to be a valid
+   * local file URI.
+   */
+  private static URI adjustLocalFileURI(URI fileUri) {
+    String uriString = fileUri.toString();
+    // Ensure the URI starts with "file:///" for absolute paths
+    if (uriString.startsWith("file:/") && !uriString.startsWith("file:///")) {
+      uriString = "file://" + uriString.substring(5);
+    }
+    return URI.create(uriString);
+  }
+
+  /**
+   * Converts a given input path or URI into a standardized URI string. This method ensures that
+   * local file paths are correctly formatted as file URIs and that URIs for different storage
+   * providers (e.g., S3, Azure, GCS) are handled appropriately.
+   *
+   * <p>If the input is a valid URI with a recognized scheme (e.g., "file", "s3", "abfs", etc.), the
+   * method returns a standardized version of the URI. If the input is not a valid URI, it treats
+   * the input as a local file path and converts it to a "file://" URI.
+   *
+   * @param inputPath the input path or URI to be standardized.
+   * @return the standardized URI string.
+   * @throws CliException if the input path has an unsupported URI scheme.
+   *     <p>Examples of input and output:
+   *     <pre>
+   * // Local File System Example:
+   * "file:/tmp/myfile"         -> "file:///tmp/myfile"
+   *
+   * // AWS S3 Example:
+   * "s3://my-bucket/my-file"   -> "s3://my-bucket/my-file"
+   *
+   * // Azure Blob Storage Example:
+   * "abfs://my-container@my-storage.dfs.core.windows.net/my-file"
+   *                          -> "abfs://my-container@my-storage.dfs.core.windows.net/my-file"
+   *
+   * // Google Cloud Storage Example:
+   * "gs://my-bucket/my-file"   -> "gs://my-bucket/my-file"
+   *
+   * // Invalid Path Example (treated as a file path):
+   * "/local/path/to/file"      -> "file:///local/path/to/file"
+   *
+   * // Unsupported Scheme Example:
+   * "ftp://example.com/file"   -> Throws BaseException with message: "Unsupported URI scheme: ftp"
+   * </pre>
+   */
+  public static String toStandardizedURIString(String inputPath) {
```

Comment:
Is this the same as `FileOperation` from the server module ? I really don't want to copy everything that we need to use in client side from the server side..... which is not fixing the tech debt, instead it's producing more tech debt. 

https://github.com/unitycatalog/unitycatalog/blob/fc76ab52aa03550d6577eca7be90004a5d2913b8/server/src/main/java/io/unitycatalog/server/persist/utils/FileOperations.java#L209-L269